srtcore/queue.cpp

```diff
@@ -604,8 +604,29 @@ void *CSndQueue::worker(void *param)
             self->m_WorkerStats.lNotReadyPop++;
 #endif /* SRT_DEBUG_SNDQ_HIGHRATE */
         }
-
+#if 0 //>>duB:debug OutPace
```

Comment:
This should allow enabling this code with `cmake . -DENFORCE_SRT_DEBUG_OUTPACE=1`
```suggestion
#if defined(SRT_DEBUG_OUTPACE)
```